ansible-playbooks/ros_install_and_set_up.yml

```diff
@@ -10,21 +10,18 @@
 - hosts: all
   gather_facts: True
   tasks:
-    - name: Install insights-client if not already installed.
+    - name: Install insights-client if not already installed
```

Comment:
```suggestion
    - name: Ensure latest insights-client is installed
```

Reply:
Cool sure, I would prefer enhancing it bit more with:
`Ensure latest version of insights-client is installed`